firstvoices/backend/urls.py

```diff
@@ -57,14 +57,14 @@
     r"ignored-characters", IgnoredCharactersViewSet, basename="ignoredcharacter"
 )
 sites_router.register(r"images", ImageViewSet, basename="image")
+sites_router.register(r"join-requests", JoinRequestViewSet, basename="joinrequest")
```

Comment:
The postman endpoints should be updated to reflect the removal of the hyphen

Reply:
I didn't change the url, only the internal "basename" used to generate the viewnames (like `api:joinrequest-detail` for reversing urls). So Postman should be good still :)